helm/templates/serviceaccount.yaml

```diff
@@ -3,5 +3,5 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: kavir
-  namespace: "{{ .Values.namespace }}"
+  namespace: "{{ .Release.Namespace }}"
```

Comment:
I don't think the namespace is even neccessary, this is usually transparently handled by Helm